common/djangoapps/third_party_auth/settings.py

```diff
@@ -38,6 +40,9 @@ def apply_settings(django_settings):
     # Where to send the user once social authentication is successful.
     django_settings.SOCIAL_AUTH_LOGIN_REDIRECT_URL = _SOCIAL_AUTH_LOGIN_REDIRECT_URL
 
+    # Adding extra key value pair in the url query string for microsoft as per request
+    django_settings.SOCIAL_AUTH_AZUREAD_OAUTH2_AUTH_EXTRA_ARGUMENTS = _SOCIAL_AUTH_AZUREAD_OAUTH2_AUTH_EXTRA_ARGUMENTS
```

Comment:
How is this setting used? Is this an existing setting in our system? Where is it used and referenced?